framework/include/mesh/MooseMesh.h

```diff
@@ -449,6 +450,14 @@ class MooseMesh : public MooseObject, public Restartable
    */
   const Moose::PatchUpdateType & getPatchUpdateStrategy() const;
 
+  /**
+   * Method for adding Relationship managers (libMesh ghosting functors) to the Mesh object. The
+   * parameter is passed in via shared_ptr. It's expected that the Mesh object will assume ownership
```

Comment:
Stale method (the body even got removed). The App will hold onto all GhostingFunctors since  libMesh doesn't currently have a mechanism to hold unto them directly. libMesh expects a reference and the lifetime needs to last for the lifetime of the libMesh objects to which they are attached.